catppuccinpalette.dtx.tera

```diff
@@ -0,0 +1,362 @@
+---
+whiskers:
+    version: "2.5.0"
+    app: "LaTeX"
+version: 1.1.0
+---
+% \iffalse meta-comment
+%
+% MIT License
+% 
+% Copyright (c) {{ now() | date(format="%Y") }} Catppuccin
+% 
+% Permission is hereby granted, free of charge, to any person obtaining a copy
+% of this software and associated documentation files (the "Software"), to deal
+% in the Software without restriction, including without limitation the rights
+% to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+% copies of the Software, and to permit persons to whom the Software is
+% furnished to do so, subject to the following conditions:
+% 
+% The above copyright notice and this permission notice shall be included in all
+% copies or substantial portions of the Software.
+% 
+% THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+% IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+% FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+% AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+% LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+% OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+% SOFTWARE.
```

Comment:
```suggestion
{{ read_file(path="LICENSE") | indent(prefix="% ", first=true, blank=true) }}
```
This (untested) should instead read your license file and indent it.

Reply:
>  don't I loose the ability of modifying the year in the license by that

Technically I don't think that is actually supposed to be updated? We don't update it in other repositories - most cases I think the license year is assumed as the *start* date (going until present).

Reply:
Works with me. Wasn't quite sure on that one.